src/backend/actions/new/new-action.ts

```diff
@@ -1,5 +1,6 @@
 import { populator } from '../../utils'
```

Comment:
How about delete action?

Reply:
I don't think we need this for delete action.

The problem which this module is supposed to solve is that AdminJS uses form data to send payload to the backend. Form data's properties are all strings (or booleans, these are actually sent as either true/false which is correct), so even if we use a `number` input in the frontend and type `123` it will be sent as a string: `"123"`.

Usually, ORMs/ODMs do the basic type conversion but it is a problem for AdminJS's `mixed` type because it saves the JSON in the same format it's sent from the frontend.

Reply:
Thanks, I got it!